### ISSUE TITLE
Pallete buttons

### ISSUE THREAD

**Issue by x2nie** [fredvs/uidesigner_ext#3]
Hi, I've played this many times until I know where the problem is.

The bug was visually the button location.
1. Before resizing:![image](https://f.cloud.github.com/assets/1678529/2393587/517b843e-a996-11e3-8a39-8a21e6abc15a.png)
The button after radiobox is combobox.

2. After form resized:
![image](https://f.cloud.github.com/assets/1678529/2393589/69746a56-a996-11e3-8c82-c4cb8cddaca3.png)

Combobox should be in the next line. this is the bug.
To reproduce the bug, just resize the form, close uidesigner_ext,  reopen, and resize again.

When I look at the code, it's fine!

The real bug is when calling gINI in the bottom (last) line of ".AfterCreated()" , which is then gINI resizes the form without triggering the palletebox to be resized. 
it turn to wrong calculation made when the pallete realign it's children.
In short words: pallete's width is larger than it's visible part. 

There is some hidden area in palette because the gINI didn't properly set the form's width .
